client/network/src/utils.rs

```diff
@@ -56,6 +56,75 @@ impl<T: Hash + Eq> LruHashSet<T> {
 		false
 	}
 }
+pub(crate) mod json_file {
```

Comment:
These modules should either be their own files or be removed. Same applies for the others.